openpathsampling/engines/trajectory.py

```diff
@@ -99,18 +99,6 @@ def reversed(self):
 
         return Trajectory([snap for snap in reversed(self)])
 
-    def prepend(self, snapshot):
-        """
-        Prepend a snapshot
-
-        Just convenience method to replace insert(0, snapshot)
-        """
-        self.insert(0, snapshot)
-        # And a generation of scientist-programmers who grew up learning
-        # "OPS trajectories are just Python lists" scream in pain when they
-        # find this after googling "python list.prepend not working".
-        # (Blame JHP. This was his doing.) ;)
```

Comment:
Alas, the end of one of my favorite comments in our code.

Still, for the better.